apps/web/src/pages/SearchRequests/ViewSearchRequestPage/components/ViewSearchRequest.tsx

```diff
@@ -259,6 +267,21 @@ export const ViewSearchRequest = ({
           data-h2-padding="base(x1)"
           data-h2-background-color="base(gray.lightest)"
         >
+          <FilterBlock
+            title={intl.formatMessage({
+              defaultMessage: "Reason for talent request",
+              id: "enffKD",
+              description: "Label for the reason for submitting the request.",
+            })}
+            content={
+              reason ? intl.formatMessage(getSearchRequestReason(reason)) : ""
+            }
+          />
```

Comment:
Less a specific ask and more a question if there is any rule of thumb for null state copy. We've added null state copy here and there but not without any specific rhythm as far as I can tell. 

So I was just wondering if this was fine for null state, so all requests submitted before this goes live. 
@BrookeCameron 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/901438e7-fcb0-4f48-884d-f861c22e43ab)

Reply:
as long as its **only** for requested submitted before this is live, then I don't see an issue with it.